### DISCUSSION TITLE
Model viewer not loading with Ajax

### DISCUSSION THREAD

**Discussion by alinoluxe** [google/model-viewer#4919] [Q&A] [answered]
I'm using wordpress as my content manager for my porftolio website with a built in theme that offers loading pages with Ajax when navigating to another page it loads the target content without reloading the current page. 

I have a blog section with a post where I showcase the model-viewer. The issue arrives that the model viewer component doesnt load correctly when you navigate from Stories page to the post  until I refresh that page, then it does. 

What could be a possible solution?

Link to video showcasing the issue -  https://www.loom.com/share/7d7be143a9064105ba874b9c84be34d3?sid=bfd73886-d073-4ddd-a378-0b87ff007154
Link to website - https://www.aovisuals.com

**Comment by elalish** [comment, marked as answer]
I don't know much about how wordpress works, but I can see that your model-viewer script (once it loads on a refresh) is in your `<head>` while all your other scripts are in the `<body>`. It seems to me you must be loading this script differently than all the rest. I also notice you're independently loading three.js, which is inefficient and may also cause conflicts.
